package.json

```diff
@@ -31,10 +31,11 @@
     "typescript": "h-joo/TypeScript#ts45-no-double-comments"
   },
   "scripts": {
-    "build": "tsc ",
+    "build": "$(yarn bin)/tsc ",
```

Comment:
I think this is not necessary. At least with `yarn build` it should find and execute any binaries from node_modules automatically.

Reply:
Good to know, done.